### PR TITLE
add main content

### PR DESCRIPTION
**Pull request by janasourav916**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,5 +2,5 @@ <h1>My First Open Source</h1>
 <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Exercitationem, esse!</p>
 <hr/>
 <main>
-
+add content to main
```
Comment (mlindia3) [inline index.html]: Please mention starting letter is caps

Comment (janasourav916) [reply index.html]: sure

**Review by mlindia3 — CHANGES_REQUESTED** [review]
Please make changes

**Review by janasourav916 — COMMENTED** [review]
Done

**Review by mlindia3 — APPROVED** [review]

**Pull request merged**